python/GafferCortexUI/ParameterisedHolderUI.py

```diff
@@ -59,6 +59,10 @@
 # NodeUI
 ##########################################################################
 
+# Supported userData entries :
+#
+# ["UI"]["showHeader"]
```

Comment:
I suspect you got the "show" prefix from the existing "showIndices" user data so there's obviously some precedent, but looking at the rest of Gaffer, I think "headerVisible" would be more in keeping. It's also more compatible with adding accessors later (setHeaderVisible() sounds better than setShowHeader()). Does that sound OK to you?